terraform-plans/templates/github/pyproject.toml.tftpl

```diff
@@ -5,19 +5,30 @@
 # - When the PR merges, the soleng-terraform bot will open a PR to the target repositories with the changes.
 
 [tool.flake8]
-ignore = ["C901", "D100", "D101", "D102", "D103", "W503", "W504"]
-exclude = ['.eggs', '.git', '.tox', '.venv', '.build', 'build', 'lib', 'report']
 max-line-length = 99
 max-doc-length = 99
 max-complexity = 10
-exclude = [".git", "__pycache__", ".tox", "build", "dist", "*.egg_info", "venv", ".venv"]
+exclude = [
+    ".git",
+    "__pycache__",
+    ".tox",
+    ".build",
+    "build",
+    "dist",
+    ".eggs",
+    "*.egg_info",
+    "venv",
+    ".venv",
+    "report",
+    "lib",
```

Comment:
Wait, are we sure we want to exclude lib?  I know I see built charms often pulling from lib; this is OK if none of our dependent charms have their own prepopulated lib directory, but are we sure of this?

I would be more comfortable if "lib" was not excluded.  I don't have a problem with the other changes here, but I think we should be linting "lib" unless we're certain it's only for modules pulled from external sources.